client/src/components/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components/macro'
+
+const MovieCard = ({ imgUrl, title, year, genre }) => {
+  return (
+    <Wrapper>
+      <img src={imgUrl} alt={title} height="562" width="375" />
+      <h2>{title}</h2>
+      <p>
+        {year} - {genre}
+      </p>
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.section`
+  background-color: var(--color-yellow);
+  border: 2px solid var(--color-lila);
+  border-radius: 10px;
+  color: #000;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0.5rem auto;
+  padding: 0.5rem;
```

Comment:
Do we really use rem for padding and margin? I thought it is better to use rem for fonts and for padding and margin px. 🤔

Reply:
Good to know! I will change this in my project. :D